spec/lib/curation_concerns/data_migration/collections_migration_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+RSpec.describe CurationConcerns::DataMigration::CollectionsMigration do
+  let(:collection1) { create(:collection, title: ['alien movies']) }
+  let(:collection2) { create(:collection, title: ['predator movies']) }
+  let(:work1) { create(:work, title: ['alien resurrection']) }
+  let(:work2) { create(:work, title: ['the predator']) }
+  let(:work3) { create(:work, title: ['alien vs. predator']) }
+
+  before do
+    collection1.members = [work1, work3]
+    collection2.members = [work2, work3]
+  end
+
+  describe '.run' do
+    it 'moves relationship from collection#members to curation_concern#member_of_collections' do
+      expect(collection1.members.to_a.size).to eq 2
+      expect(collection2.members.to_a.size).to eq 2
+      expect(work1.member_of_collections.to_a.size).to eq 0
+      expect(work2.member_of_collections.to_a.size).to eq 0
+      expect(work3.member_of_collections.to_a.size).to eq 0
+      described_class.run
+      expect(collection1.reload.members.to_a.size).to eq 0
+      expect(collection2.reload.members.to_a.size).to eq 0
+      expect(work1.reload.member_of_collections.to_a.size).to eq 1
```

Comment:
Can you check the ids of the collections here just to make sure the works stay in the right collections?